### ISSUE TITLE
¿Hidroelectrica es polimorfica?

### ISSUE THREAD

**Issue by LucasZibell** [tomi-iglesias/tpAtomico#2]
Les dejo una preguntan para que piensen, la `centralHidroelectrica` es polimorfica con las otras ciudades? Particularmente con `contamina`? Y habiendo dicho eso, la segunda pregunta seria: Springfield podria tener una central hidroelectrica o tendriamos un problema?

**Comment by tomi-iglesias** [comment]
No, no es polimorfica porque no tenemos esa funcion, de contamina. No podria tenerla porque justamente, habria un problema en el caso de que pregunte si son contaminantes o no.
